website/docs/cloud-docs/workspaces/settings/notifications.mdx

```diff
@@ -33,30 +33,26 @@ To start receiving notifications for a workspace's runs:
+  - For Microsoft Teams notifications, refer to Microsoft's documentation on [how to create an incoming webhook](https://docs.microsoft.com/en-us/microsoftteams/platform/webhooks-and-connectors/how-to/add-incoming-webhook).
+- **Token** Optional; only available for generic webhooks. An arbitrary secret string that Terraform Cloud will use to sign its notification webhooks. Refer to [Notification Authenticity][inpage-hmac] for details. This setting is encrypted for storage, so you cannot view it after saving.
 
-* **Name:** An arbitrary display name to use for this notification configuration.
+- **Email Recipients** Only available for emails. Select the users from the workspace's organization to notify.
 
-* **Webhook URL** _(only available for generic and slack webhooks):_ The destination URL to send the webhook payload to. This URL must accept HTTP or HTTPS `POST` requests, and should be able to do something useful with the chosen payload type.
-  * If you are sending notifications to Slack, see Slack's documentation on [how to create an incoming webhook](https://api.slack.com/messaging/webhooks#create_a_webhook).
-
-* **Token** _(optional; only available for generic webhooks):_ An arbitrary secret string, which Terraform Cloud will use to sign its notification webhooks. ([See "Notification Authenticity" below][inpage-hmac].) This setting is encrypted for storage, and cannot be viewed after it is saved.
-
-* **Email Recipients** _(only available for emails):_ Select the users from the workspace's organization to notify.
-
-* **Triggers:** Whether to notify this destination for all events, or only for selected events. The following specific events are available:
-  * **Created**: When a run is created and enters the "Pending" state.
-  * **Planning**: When a run acquires the lock and starts to execute.
-  * **Needs Attention**: Human decision required. When a plan has changes and is not auto-applied, or requires a policy override.
-  * **Applying**: When a run begins the apply stage, after a plan is confirmed or auto-applied.
-  * **Completed**: When a run has completed on a happy path and can't go any further.
-  * **Errored**: When a run has terminated early due to error or cancellation.
+- **Triggers:** Whether to notify this destination for all events, or only for selected events. The following specific events are available:
+  - **Created**: When a run is created and enters the **Pending** state.
+  - **Planning**: When a run acquires the lock and starts to execute.
+  - **Needs Attention**: Human decision required. When a plan has changes and is not auto-applied, or requires a policy override.
+  - **Applying**: When a run begins the apply stage, after a plan is confirmed or auto-applied.
+  - **Completed**: When a run has completed successfully.
+  - **Errored**: When a run has terminated early due to error or cancellation.
```

Comment:
I'm so sorry I missed this last time. Should **Pending** be in this list separately? Is it ever in Created, or does it go directly to pending?

```suggestion
  - **Created**: The run is created and enters the **Pending** state.
  - **Planning**: The run acquires the lock and starts to execute.
  - **Needs Attention**: Human decision required. When a plan has changes and is not auto-applied, or requires a policy override.
  - **Applying**: The run begins the apply stage, after a plan is confirmed or auto-applied.
  - **Completed**: The run has completed successfully.
  - **Errored**: The run has terminated early due to error or cancellation.
```

Reply:
Oh, good question!  There isn't actually created state for a run. As soon as a run is created, it is in the "pending" state.

The notification is called "created" (you'll get something in your email, or slack, like "hey, a new run has been created"). That notification gets _generated_ when a run enters the pending state...i.e., as soon as the run gets created.

Hopefully that makes sense...I think it is confusing how we mix notification triggers and run states here, now that you point it out. Maybe something like `The run is created, and is queued`, what do you think @laurapacilio ?

Reply:
yeah i think I like that! I don't know I know we have other mentions of state elsewhere but this is the only one that confused me. I think that's a reasonable edit!